### DISCUSSION TITLE
How to change a window?

### DISCUSSION THREAD

**Discussion by bimawa** [kovidgoyal/kitty#4539] [Q&A] [answered]
When I create new session by ctrl+shift+enter, I can't found any docs by how to change between windows of session from keyboard? Only mouse?
Maybe we have a function to map it?

**Comment by page-down** [comment, marked as answer]
You are creating a kitty window and there are many default shortcuts to switch focus. For configuration, please refer to the documentation and the mappable actions section.

https://sw.kovidgoyal.net/kitty/overview/#windows

**Reply by bimawa** [reply]
Yeaa, thanks! That what I was searching... )
